src/index.cpp

```diff
@@ -49,8 +49,11 @@
 #include "tooltip.h"
 #include "utf8.h"
 
-#define MAX_ITEMS_BEFORE_MULTIPAGE_INDEX 200
-#define MAX_ITEMS_BEFORE_QUICK_INDEX 30
+//#define MAX_ITEMS_BEFORE_MULTIPAGE_INDEX 2000
+//#define MAX_ITEMS_BEFORE_QUICK_INDEX 30
+
+#define MAX_ITEMS_BEFORE_MULTIPAGE_INDEX (int) Config_getInt(MAX_ITEMS_BEFORE_MULTIPAGE_INDEX)
+#define MAX_ITEMS_BEFORE_QUICK_INDEX (int) Config_getInt(MAX_ITEMS_BEFORE_QUICK_INDEX)
```

Comment:
Why using `(int)` here? The `ConfigGetInt` returns an `int`.

Reply:
It solved some compiler warnings about comparing unsigned long to int.

Reply:
I thought it solved some compiler warnings about comparing int to unsigned long, that turned out to be unrelated.